website/content/docs/usage.md

```diff
@@ -4,19 +4,19 @@ description: Run a Virtual Kubelet inside or outside of your Kubernetes cluster
 weight: 2
 ---
 
-Virtual Kubelet is run via the `virtual-kubelet` command-line tool (documented [below](#virtual-kubelet-cli)). You can run Virtual Kubelet either [outside](#outside-k8s) or [inside](#inside-k8s) of a Kubernetes cluster.
+You can Virtual Kubelet either [outside](#outside-k8s) or [inside](#inside-k8s) of a Kubernetes cluster using the [`virtual-kubelet`](#virtual-kubelet-cli) command-line tool. If you run Kubernetes inside of a Kubernetes cluster, you can also deploy it using [Helm](#helm).
```

Comment:
What does inside vs. outside mean in this context?

Reply:
Oh I see, maybe we should say locally instead?

Reply:
This language is borrowed from the README so I'm off the hook 😄 I'll take a look and see if I can make that more clear.

Reply:
This language is basically borrowed 1:1 from the README. I definitely agree that this needs clarification. I'll file an issue and address that in another PR.